### ISSUE TITLE
Copy-pasting multiline text to MessageBlock doubles the newlines

### ISSUE THREAD

**Issue by sk7725** [Anuken/Mindustry#3162]
**Platform**: *Windows*

**Build**: *10853*

**Issue**: Copy a multiline text:
```
Behold.
The cat.
And his wonder router.
```
Paste it in the MessageBlock:
```
Behold.

The cat.

And his wonder router.
```
The newlines double, adding extra newlines at line 2 and 4 in the example.

**Steps to reproduce**: *See above for an example.*

**Link(s) to mod(s) used**: *null*

**Save file**: *Issue is from TextArea.java as far as I know.*

**Crash report**: *null*

---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  
- [X] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

**Comment by ThePythonGuy3** [comment]
`Save file: Issue is from TextArea.java as far as I know.`
*Good job* 👏

**Comment by DeltaNedas** [comment]
simple, just dont add carriage returns

**Comment by Anuken** [comment]
fixed

**Comment by sk7725** [comment]
> simple, just dont add carriage returns

@DeltaNedas what the heck are carriage returns

**Comment by Anuken** [comment]
things on windows that shouldn't exist

**Comment by DeltaNedas** [comment]
a character that indicates to move the typewriters carriage to the left.
these were usually spammed like \r\r\r\r\r\r before a newline to ensure it moved all the way
then \r\n in DOS
in UNIX and all normal oses lines are terminated by a single \n
tl;dr typewriter
